css-grid-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+.container {
+  margin: auto;
+  width: 1000px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+}
+
+.column-full {
+  display: flex;
+  align-self: flex-start;
+}
+
+.column-one-third {
+  margin: 5px;
```

Comment:
If you're going to use an exact width like this, this margin will break your content if you ever set the row to `flex-wrap: wrap`. This should be removed.

Reply:
Got it, Thank you Cody!

I will adjust my code for all the changes